src/main/java/com/dev/cinema/model/Ticket.java

```diff
@@ -0,0 +1,64 @@
+package com.dev.cinema.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "ticket")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.EAGER, cascade = CascadeType.PERSIST)
```

Comment:
```suggestion
    @ManyToOne(cascade = CascadeType.PERSIST)
```
тут fetch = FetchType.EAGER по дефолту

Reply:
+